app/models/manageiq/providers/openstack/network_manager/cloud_subnet.rb

```diff
@@ -121,7 +126,10 @@ def self.params_for_create(ems)
   end
 
   def self.raw_create_cloud_subnet(ext_management_system, options)
-    cloud_tenant = options.delete(:cloud_tenant)
+    cloud_network = CloudNetwork.find_by(:id => options[:cloud_network_id])
+    options[:network_id] = cloud_network.ems_ref
```

Comment:
Added the change discussed here: https://github.com/ManageIQ/manageiq-ui-classic/pull/7734#discussion_r633543299

Reply:
@agrare looks like this change broke a test case here: https://github.com/ManageIQ/manageiq-providers-openstack/blob/master/spec/models/manageiq/providers/openstack/network_manager/cloud_subnet_refresh_spec.rb#L85

Reply:
You'll have to fix that test then